src/xrpld/app/misc/CredentialHelpers.cpp

```diff
@@ -272,6 +318,48 @@ checkArray(STArray const& credentials, unsigned maxSize, beast::Journal j)
 
 }  // namespace credentials
 
+TER
+verifyValidDomain(
+    ApplyView& view,
+    AccountID const& account,
+    uint256 domainID,
+    beast::Journal j)
+{
+    auto const slePD = view.read(keylet::permissionedDomain(domainID));
+    if (!slePD || !slePD->isFieldPresent(sfAcceptedCredentials))
```

Comment:
Why do we need to check for `AcceptedCredentials`? It is a required field.

Reply:
thanks, removed